### PR TITLE
Change specification and xsd-fu licences from [L]GPL to BSD 2-clause

### PR DESCRIPTION
**Pull request by rleigh-dundee**
See https://trello.com/c/laC87bbC/45-bsd-licensing-consistency

No code changes; this changes all copyright boilerplate to BSD in these components, adding it where missing.

### CODE REVIEW

File: components/specification/transforms/2007-06-to-2008-02.xsl [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2007-06-to-2008-02.xsl]: The original list of copyright holders should be restored here.

File: components/specification/transforms/2010-06-to-2003-FC.xsl [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2010-06-to-2003-FC.xsl]: Same as above. The original list of copyright holders should be restored here.

File: components/specification/transforms/2010-06-to-2008-02.xsl [hunk 3]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2010-06-to-2008-02.xsl]: Same as above

File: components/specification/transforms/2011-06-to-2010-06.xsl [hunk 4]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2011-06-to-2010-06.xsl]: Same as above

File: components/specification/transforms/2012-06-to-2011-06.xsl [hunk 5]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2012-06-to-2011-06.xsl]: Same as above

File: components/specification/transforms/2013-06-to-2012-06.xsl [hunk 6]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2013-06-to-2012-06.xsl]: Same as above

File: components/specification/transforms/2013-06-to-2015-01.xsl [hunk 7]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2013-06-to-2015-01.xsl]: Same as above

File: components/specification/transforms/2015-01-to-2013-06.xsl [hunk 8]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2015-01-to-2013-06.xsl]: Same as above

File: components/specification/transforms/2015-01-to-2016-06.xsl [hunk 9]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2015-01-to-2016-06.xsl]: Same as above

File: components/specification/transforms/2016-06-to-2015-01.xsl [hunk 10]
```diff
@@ -1,29 +1,35 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/2016-06-to-2015-01.xsl]: Same as above

File: components/specification/transforms/internal/OME-CA2OME.xslt [hunk 11]
```diff
@@ -1,30 +1,35 @@
...
-#    Lesser General Public License for more details.
-#
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  BSD implementations of Bio-Formats readers and writers
```
Comment (sbesson) [inline components/specification/transforms/internal/OME-CA2OME.xslt]: OME Data Model transforms ? Not sure how these internal transforms are used though

File: components/specification/transforms/ome-transforms.xml [hunk 12]
```diff
@@ -1,35 +1,35 @@
...
-	#    You should have received a copy of the GNU Lesser General Public
-	#    License along with this library; if not, write to the Free Software
-	#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-	#
-	#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/ome-transforms.xml]: Same as above. Restore the original copyright holders list

File: components/specification/transforms/units-conversion.xsl [hunk 13]
```diff
@@ -1,29 +1,36 @@
...
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
---><!--
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2014 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/units-conversion.xsl]: Same as above.

File: components/specification/transforms/util/write-transforms.py [hunk 14]
```diff
@@ -139,30 +150,36 @@ def print_path(to_schema):
...
-	#    You should have received a copy of the GNU Lesser General Public
-	#    License along with this library; if not, write to the Free Software
-	#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-	#
-	#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  OME Data Model transforms
+  %%
+  Copyright (C) 2005 - 2016 Open Microscopy Environment:
```
Comment (sbesson) [inline components/specification/transforms/util/write-transforms.py]: Also restore the original copyright holders list

File: components/specification/transforms/internal/OME2OME-CA.xslt [hunk 15]
```diff
@@ -1,30 +1,35 @@
...
-#    Lesser General Public License for more details.
-#
-#    You should have received a copy of the GNU Lesser General Public
-#    License along with this library; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-#
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+  #%L
+  BSD implementations of Bio-Formats readers and writers
```
Comment (sbesson) [inline components/specification/transforms/internal/OME2OME-CA.xslt]: Also OME Data Model transforms?

**Comment by rleigh-dundee** [comment]
@sbesson The copyright holders should be fixed up now; note I reformatted them to match the style of the other licences in the rest of these components and elsewhere.

**Comment by sbesson** [comment]
Thanks @rleigh-dundee, got the green flag from @chris-allan on this PR as well. Including this relicensing for 5.2.0.

**Pull request merged**